install/prereqs/index.rst

```diff
@@ -50,8 +50,8 @@ The LSST Science Pipelines require Python 3.6 or newer.
 Both the :doc:`newinstall.sh <../newinstall>` and :doc:`lsstsw <../lsstsw>`\ -based installation methods provide dedicated Miniconda environments pre-loaded with Python dependencies.
 If you opt to use your own Python, you can re-create the default Python environment made by :command:`newinstall.sh` and ``lsstsw`` with these Conda environments:
 
-- `macOS <https://github.com/lsst/lsstsw/blob/master/etc/conda3_packages-osx-64.txt>`_.
-- `Linux <https://github.com/lsst/lsstsw/blob/master/etc/conda3_packages-linux-64.txt>`_.
+- `macOS <https://github.com/lsst/lsstsw/blob/16.0/etc/conda3_packages-osx-64.txt>`_.
+- `Linux <https://github.com/lsst/lsstsw/blob/16.0/etc/conda3_packages-linux-64.txt>`_.
```

Comment:
Being strictly accurate, these package lists are *not* what you get if you use `newinstall.sh`.

Specifically, version 16.0 of `newinstall.sh` [pulls in package lists from lsstsw `10a4fa6`](https://github.com/lsst/lsst/blob/16.0/scripts/newinstall.sh#L42).

However, the SHA1 corresponding to the package lists  tagged `16.0` is actually [`dfec933`](https://github.com/lsst/lsstsw/blob/16.0/etc/conda3_packages-osx-64.txt).

The differences are minor (basically, the addition of the h5py package), but strictly speaking we are not testing against the environment you describe above.

Reply:
Please us `10a4fa6` as the git ref as this was/is the hard-coded value used for the tarball build.